.github/workflows/tests.yml

```diff
@@ -0,0 +1,97 @@
+name: BlueChi Tests
+
+on:
+  push:
+    branches: [main, hirte-*]
+  pull_request:
+    branches: [main, hirte-*]
+  workflow_dispatch:
+
+jobs:
+  unit-tests:
+    uses: ./.github/workflows/unit-tests.yml
```

Comment:
If this PR is accepted, we should change this in a follow-up (or here directly) to `eclipse-bluechi/bluechi/.github/workflows/unit-tests.yml@main` (and for the integration tests) to pin the workflow version here.